src/script/src/manager.rs

```diff
@@ -90,8 +95,12 @@ impl ScriptManager {
             .context(ExecutePythonSnafu { name })
     }
 
-    async fn try_find_script_and_compile(&self, name: &str) -> Result<Option<Arc<PyScript>>> {
-        let script = self.table.find_script_by_name(name).await?;
+    async fn try_find_script_and_compile(
+        &self,
+        schema: &str,
+        name: &str,
+    ) -> Result<Option<Arc<PyScript>>> {
+        let script = self.table.find_script_by_name(name, schema).await?;
```

Comment:
Looks like we use pass the schema first?
```suggestion
        let script = self.table.find_script_by_name(schema, name).await?;
```

Reply:
Yes